1.0/docs/start/psk/setting-up.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+type: start
+shortname: Start
+title: Setting up
+subtitle: Polymer Starter Kit
+---
+
+## Setting up the Polymer Starter Kit
```

Comment:
:heavy_check_mark: removed all gerunds from headings and filenames

:heavy_check_mark: split page into more `h2` sections